### DISCUSSION TITLE
Saat memvalidasi halaman AMP muncul error "The mandatory text inside tag 'head > style[amp-boilerplate]' is missing or incorrect."

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#191] [Q&A] [answered]
Kang, Saat saya memvalidasi halaman AMP muncul error "The mandatory text inside tag 'head > style[amp-boilerplate]' is missing or incorrect.". Padahal penulisan kode normal semua. sesuai rules

` <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>`

Setelah saya coba googling error ini berasal dari minify. dan saya coba matikan extension minify nya, error amp tersebut langsung hilang.

Pertanyaannya:
Apakah ada solusi membereskan error ini tanpa mematikan extensionnya minify?
Atau membuat kondisional pada halaman non amp extension minify hidup dan pada halaman amp extension minify dimatikan?

**Comment by taufik-nurrohman** [comment, marked as answer]
~~~ php
if ($site->is('mobile')) {
    Hook::let('content', "x\\minify");
}
~~~
